projects/plugins/jetpack/extensions/blocks/ai-assistant/extensions/jetpack-form-block-ai-composition/index.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * External dependencies
+ */
+import { BlockControls } from '@wordpress/block-editor';
+import { getBlockType } from '@wordpress/blocks';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { select } from '@wordpress/data';
+import { addFilter } from '@wordpress/hooks';
+/*
+ * Internal dependencies
+ */
+import { AI_Assistant_Initial_State } from '../../hooks/use-ai-feature';
```

Comment:
Why we're exporting something related with initial state from an `hook`? It's sounds unrelated for me.

Reply:
We may not export that, but it seems unrelated to this PR. We can change that if we want.

Regarding the export of the initial state, it's used for the hook to access the data. However, there may be a more efficient method of implementation that we can explore.

Reply:
> but it seems unrelated to this PR

Not exactly, in this PR it is being imported from the hook. IMO we shouldn't import this from an unrelated place.

Reply:
Let's do that in a follow-up. The data provided by the hook [here](https://github.com/Automattic/jetpack/blob/3a11b4f720abe750d43ee6384cd86adb46d75efa/projects/plugins/jetpack/extensions/blocks/ai-assistant/hooks/use-ai-feature/index.ts#L36-L45) is optimal for this purpose.

Feel free to address it, or create the issue. I want to keep the PR simple.